primes.rb

```diff
@@ -1,6 +1,22 @@
 class Primes
-  def self.sum_to(limit = 100)
-    # TODO - add your prime number solution here...
-    "I'm working on it!"
+  def self.sum_to(limit)
+    # TODO - add your prime number solution here... 
+    require 'prime'
+    return Prime.each(limit).inject(0, :+)
   end
-end
\ No newline at end of file
+end
+
+
+# working on the solution how to make this code work in app
+=begin
+class Puzzle
+  def mult()
+  (1...1000).select do |x|
+    x % 3 == 0 || x % 5 == 0
+    end
+  end
+  def self.sum_mult(mult)
+  add = mult.inject(&:+)
```

Comment:
Looks like you were on the right lines here. I think `mult` needed to be a class method as well, otherwise you'd need to instantiate an object to call it on. This is the kind of thing we'll go over next week.